base/docs/helpdb/Base.jl

```diff
@@ -351,79 +351,6 @@ See [`RoundingMode`](:obj:`RoundingMode`) for available rounding modes.
 Float32
 
 """
-    Mmap.mmap(io::Union{IOStream,AbstractString,Mmap.AnonymousMmap}[, type::Type{Array{T,N}}, dims, offset]; grow::Bool=true, shared::Bool=true)
```

Comment:
IO doesn't include AbstractString - could document that possibility with its own docstring, or go back to something like the Union. Are there methods implemented for arbitrary other IO subtypes?